mysql-test/suite/innodb/r/temp_table_encrypt_keyring_file.result

```diff
@@ -9,19 +9,20 @@ Pattern found.
 DROP TABLE t04;
 SET GLOBAL innodb_temp_tablespace_encrypt = ON;
 CREATE TEMPORARY TABLE t02 (a INT) ENGINE=InnoDB ENCRYPTION='Y';
-ERROR HY000: InnoDB: Unsupported encryption option for temporary tables.
```

Comment:
This is not review, just a comment. `temp_table_encrypt_keyring_vault.result` should also be re-recorded I guess.

Reply:
right, missed it

Reply:
this test cannot be run as of now because of some dependency with key_ring vault plugin. Robert promised me to ping me when the test setup is fixed. Will fix that when it is ready